Dockerfile

```diff
@@ -98,6 +98,7 @@ RUN mkdir -p /opt/code/localstack/localstack && \
     ln -s /usr/lib/localstack /opt/code/localstack/localstack/infra && \
     mkdir /tmp/localstack && \
     chmod -R 777 /tmp/localstack && \
+    touch /tmp/localstack/.marker && \
```

Comment:
q: Does the marker file always stay? There's no reboot in docker, but is there any timed cleanup of tmp files in our debian base image?